### PR TITLE
NetworkMembership.Nominate working: 1

### PR DESCRIPTION
**Pull request by kartikaysaxena**
Rebased current branch for a base commit which would be utilised for rebasing with other PRs

### CODE REVIEW

**Review by vatsa287 — DISMISSED** [review]

File: demo/go.mod [hunk 1]
```diff
@@ -0,0 +1,34 @@
+module github.com/kartikaysaxena/cord.go/demo
```
Comment (vatsa287) [inline demo/go.mod]: Please change this to have under `dhiway`

File: demo/main.go [hunk 2]
```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	utils "github.com/kartikaysaxena/cord.go/packages/utils/src"
```
Comment (vatsa287) [inline demo/main.go]: Here too.

File: demo/main.go [hunk 3]
```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	utils "github.com/dhiway/cord.go/packages/utils/src"
+	gsrpc "github.com/kartikaysaxena/substrateinterface"
```
Comment (vatsa287) [inline demo/main.go:8]: Missed this?

Comment (vatsa287) [reply demo/main.go:8]: `Substrate-Interface` is stil not updated.

Comment (kartikaysaxena) [reply demo/main.go:8]: missed it, just updated

**Review by vatsa287 — APPROVED** [review]

**Pull request merged**